pkg/docker/docker.go

```diff
@@ -359,6 +365,30 @@ func GetContainerRegistryAuthStr(jsonKeyFile string) (string, error) {
 	return containerRegistryAuthStr, nil
 }
 
+func PruneResources(client *dockerclient.Client) error {
```

Comment:
is this just a global prune? like it will prune images for images from different pods?